tests/modules/site_settings/resources/test_read_configurations.py

```diff
@@ -84,6 +82,7 @@ def test_alter_edm_settings(flask_app_client, admin_user):
     )
 
 
+# TODO sort this out as part of DEX 1306
 @pytest.mark.skipif(extension_unavailable('edm'), reason='EDM extension disabled')
```

Comment:
Does this `skipif` need to go?

Reply:
It will do but as per the comment above this needs rework as part of DEX 1306 anyway.